babar-report/src/IO.vue

```diff
@@ -11,7 +11,7 @@
       <b-container fluid>
         <PlotTimeSeries title="Network bytes received/transmitted / sec" yAxis="bytes/sec" :series="series.netIO" />
         <div class="explanation">
-          This graph shows the total number of bytes of data transmitted or received by all non-virtual interfaces by the entire process-tree as reported by the <kbd>/prod/net/dev</kbd> file.<br>
+          This graph shows the total number of bytes of data transmitted or received by all non-virtual interfaces by the entire process-tree as reported by the <kbd>/proc/net/dev</kbd> file.<br>
```

Comment:
I may have missed this one on the first pass, but if I am not mistaken it is not the network usage of the process tree but the one of the entire system (the host OS)?